src/client/datascience/interactive-ipynb/nativeEditor.ts

```diff
@@ -101,6 +101,10 @@ export class NativeEditor extends InteractiveBase implements INotebookEditor {
         return this.viewState.active;
     }
 
+    public get selection() {
```

Comment:
Feels unnecessary to add more types into our editor when we do'nt need this and don't support it.
Why not just use the VSC Notebook API directly (instead of making our stuff look like its a VS Code notebook)